federation/pkg/kubefed/unjoin.go

```diff
@@ -88,7 +86,7 @@ func (u *unjoinFederation) Run(f cmdutil.Factory, cmdOut, cmdErr io.Writer, conf
 	// federation host cluster. See join_federation.go for details.
 	hostFactory := config.HostFactory(u.commonOptions.Host, u.commonOptions.Kubeconfig)
 	err = deleteSecret(hostFactory, cluster.Spec.SecretRef.Name, u.commonOptions.FederationSystemNamespace)
-	if isNotFound(err) {
+	if util.IsNotFound(err) {
```

Comment:
(no action required) I'm not sure why the secret not being found is grounds for a warning message, but the cluster doesn't warrant the same behavior.